core/primitives.go

```diff
@@ -605,13 +605,15 @@ func (array *PdfObjectArray) GetAsFloat64Slice() ([]float64, error) {
 }
 
 // Merge merges in key/values from another dictionary. Overwriting if has same keys.
-func (d *PdfObjectDictionary) Merge(another *PdfObjectDictionary) {
+func (d *PdfObjectDictionary) Merge(another *PdfObjectDictionary) *PdfObjectDictionary {
```

Comment:
Clarify the documentation that it mutates `d` and returns `d` for convenience so that it can be chained.

This changes the function signature, but I guess it does not have any risk (i.e. not a breaking change) since it did not return anything before.